src/main/proto/service.proto

```diff
@@ -24,18 +24,22 @@ option java_outer_classname = "ServiceProto";
 
 service FinApp {
 
-  rpc CreateCustomer(Customer) returns (.google.protobuf.Empty) {}
+  rpc CreateCustomer(Customer) returns (CustomerResponse) {}
```

Comment:
I think it would be better to rename to:
CreateCustomerResponse
CreateAccountResponse
CreateCustomerRoleResponse

I'd expect a message that is called CustomerResponse to have a customer response...